src/Entity/Comment.php

```diff
@@ -0,0 +1,120 @@
+<?php
+
+namespace App\Entity;
+
+use App\Repository\CommentRepository;
+use Doctrine\ORM\Mapping as ORM;
+use Symfony\Component\Validator\Constraints as Assert;
+
+/**
+ * @ORM\Entity(repositoryClass=CommentRepository::class)
+ */
+class Comment
+{
+    /**
+     * @ORM\Id()
+     * @ORM\GeneratedValue()
+     * @ORM\Column(type="integer")
+     */
+    private $id;
+
+    /**
+     * @ORM\Column(type="string", length=255)
+     * @Assert\NotBlank()
+     */
+    private $content;
+
+    /**
+     * @ORM\Column(type="datetime")
+     */
+    private $dateCreation;
```

Comment:
Pour faire le relou je dirais d'utiliser un nom générique et conventionnel genre `createdAt`

```suggestion 
     * @ORM\Column(type="datetime_immutable")
     */
    private $createdAt;
```

J'aurais tendance à te dire aussi de mettre `datetime_immutable` la date de creation ne devrait jamais bouger. Je t'invite à lire https://fr.wikipedia.org/wiki/Objet_immuable

Reply:
modifié